Oddities.WinHelp/Topics/Paragraph.cs

```diff
@@ -13,7 +13,7 @@ public enum ParagraphRecordType : byte
 }
 
 [PublicAPI]
-public struct Paragraph
+public record struct Paragraph
```

Comment:
Since this particular struct has private fields, it doesn't make much sense in converting it to a `record struct`. Let's leave it as it was before.